src/style.css

```diff
@@ -0,0 +1,31 @@
+* {
+  margin: 0;
+  padding: 0;
```

Comment:
Don't reset paddings and margins in * selector, reset them for tags you are using